Makefile.pre.in

```diff
@@ -1398,7 +1398,7 @@ Modules/getpath.o: $(srcdir)/Modules/getpath.c Python/frozen_modules/getpath.h M
 Programs/python.o: $(srcdir)/Programs/python.c
 	$(CC) -c $(PY_CORE_CFLAGS) -o $@ $(srcdir)/Programs/python.c
 
-Programs/_testembed.o: $(srcdir)/Programs/_testembed.c Programs/test_frozenmain.h
+Programs/_testembed.o: $(srcdir)/Programs/_testembed.c Programs/test_frozenmain.h $(srcdir)/Include/internal/pycore_initconfig.h $(srcdir)/Include/internal/pycore_runtime.h $(srcdir)/Include/internal/pycore_import.h
```

Comment:
`_testembed.c` includes `Python.h`, so I'd think that `Programs/_testembed.o` should depend on all of `$(PYTHON_HEADERS)`.

That's a little bit tricky because PYTHON_HEADERS is defined later on, so we'd either need to move up the definition (probably a good idea) or add the dependency later on like:

https://github.com/python/cpython/blob/191531f352ce387a2d3a61544fb6feefab754d4a/Makefile.pre.in#L1907

Reply:
Done